pkg/infinispan/cluster.go

```diff
@@ -117,6 +118,22 @@ func (c Cluster) GetClusterMembers(podName string) ([]string, error) {
 	return health.ClusterHealth.Nodes, nil
 }
 
+func (c Cluster) GetSitesView(podName string) (map[string]bool, error) {
+	cacheManagerInfo, err := c.GetCacheManagerInfo(consts.DefaultCacheManagerName, podName)
+	if err != nil {
+		return nil, err
+	}
+	siteViews := cacheManagerInfo["sites_view"]
+	if siteViews == nil {
+		return nil, fmt.Errorf("getting Cross-Site view is not supported")
```

Comment:
@dmvolod When would you hit this error? Is it more of a case that cross-site is not enabled so it's not possible to get an xsite view?

Reply:
This problem can be occurred when someone will try to use current operator version with Infinispan server image version lower than 12.1.0 release. It's not supported in downstream but anyone can provide own image version.

Reply:
Thanks. I think the error should just say that. If it only occurs under that condition.